packages/sdk/examples/notebook.ts

```diff
@@ -0,0 +1,66 @@
+// %% Notebook for Giselle SDK
```

Comment:
@toyamarinyon 

Could you please tell me how to run `examples/notebook.ts`?

I tried the following commands, but both resulted in errors:

```
npx ts-node examples/notebook.ts
```

```
npx tsc examples/notebook.ts
```

Reply:
@shige Sorry, it a test file. I've remove e0b3667650326362c90f4902bcc734da9a5b9bea